### DISCUSSION TITLE
Possible to Associate Multiple Programs with File Type?

### DISCUSSION THREAD

**Discussion by AntumDeluge** [d2phap/ImageGlass#784] [General] [answered]
### System information:
 - Windows OS version: Windows 10 64-bit
 - ImageGlass version: 7.6.4.30

### Questions
Is it possible to associate more than 1 app with a file type in **Settings -> Edit**? It doesn't appear to be so.

In this screenshot, I have an entry to open SVG files in **Inkscape**, but I also want to be able to open them in **Inkview** (which is an image preview program included with Inkscape for previewing SVG files with scaling support).
![ImageGlass-Settings-Edit](https://user-images.githubusercontent.com/3631473/82846060-e4904100-9e9b-11ea-84ca-9b98abf3e8f5.png)

**Comment by d2phap** [comment, marked as answer]
I don't think it's possible.
However, you can press "D" to use "Open with..." feature
